src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub mod driver;
 pub mod frame;
 pub mod master;
 
-pub use crate::frame::{checksum, classic_checksum, PID};
+pub use crate::frame::{checksum, classic_checksum, Frame, PID};
```

Comment:
A small fix to provide and easier import path. Noticed while starting to upgrade dependent crates.